### ISSUE TITLE
Fix restricted cookie mode

### ISSUE THREAD

**Issue by leesbian** [mageplus/mageplus#15]
If you enable the restricted cookie mode (which presents a banner asking if you'll allow or disallow cookies), Magento actually sets 2 cookies (ZDEDebuggerPresent, frontend) before asking permission to use cookies

That's kinda wrong

**Comment by LeeSaferite** [comment]
Isn't ZDEDebuggerPresent being set by the Zend Debugger?  

That would be outside the scope of Magento.

**Comment by leesbian** [comment]
yep - just tried a local PHP (but non-Magento) site and I still get the cookie set

Issue with frontend cookie still exists
